tools/ConvParallelProbe/gemm_bench.py

```
@@ -0,0 +1,137 @@
+
+
+def run_probe(probe, m, k, n, maxdop, reps, flag, repeat):
+    """Return the min min_ms over `repeat` probe invocations for one flag value."""
+    env = dict(os.environ)
+    env["AIDOTNET_DISABLE_GPU"] = "1"          # no OpenCL init stealing cores
+    env["AIDOTNET_GEMM_SMALLM_TILING"] = str(flag)
+    cmd = ["dotnet", probe, "--gemm", "--m", str(m), "--k", str(k), "--n", str(n),
+           "--reps", str(reps), "--maxdop", str(maxdop)]
+    best = None
+    for _ in range(repeat):
+        out = subprocess.run(cmd, env=env, capture_output=True, text=True, timeout=900)
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)